src/LanguageSwitcher/style.js

```diff
@@ -0,0 +1,28 @@
+import styled from '@emotion/styled'
+import { theme } from '~/utils/styles'
+
+export const LanguageSwitcherContainer = styled.div`
+  display: inline-flex;
+
+  p {
+    margin-bottom: 0px !important;
+  }
+
+  p.current {
+    font-weight: bold;
+  }
+
+  .Header__LanguageSwitcher-button {
+    text-decoration: none;
+    color: black;
+    font-family: ${theme.fontFamily.secondary};
+    line-height: 26px;
+    font-style: normal;
+    font-size: ${theme.fonts.heading6};
+
+    @media (min-width: ${theme.breakpoints.md}) {
+      font-size: ${theme.fonts.heading5};
+      line-height: 34px;
```

Comment:
Do not remove the line-height but for future reference, never use line-height from Figma. Those value are always off. Also do not set line-height in px but in that format: 1.1, 1.2 and so on

Reply:
Ok thank you for letting me know